scripts/compute_mappings/compute_ancestor_mapping.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+# coding: utf-8
+# flake8: noqa
+
+# # Ontology ancestors mapping
+# This notebook can be used to compute mappings between nodes in an ontology files and their ancestors.
+# It will:
+# 1. Load the required ontology files (human, mouse, UBERON)
+# 1. For `human` and `mouse`, start from each node available in the ontology
+# 1. for `UBERON`, only start from a defined, hardcoded subset of leaf nodes which is relevant for our datasets
+# 1. Recursively compute ancestors using the `is_a` relationship and `part_of` restriction (defined as `BFO_0000050`)
+# 1. Create mappings from each node to its ancestors
+# 1. Write a json file with these mappings (a dictionary of lists)
+
+# In[1]:
+
+
+from owlready2 import get_ontology  # noqa
+import json
+import yaml
+
+
+# In[2]:
+
+
+def get_ancestors(class_name):
+    z = onto.search_one(iri=f"http://purl.obolibrary.org/obo/{class_name}")
```

Comment:
where does `onto` come from?

Reply:
It was taking it from a global declared after this function declaration, which I agree is pretty confusing. I refactored the script to clean this part up